src/media/js/mobilenetwork.js

```diff
@@ -129,6 +105,11 @@ define('mobilenetwork',
             20: 'america_movil'
         },
 
+        // Jordan
+        416: {
+            77: 'orange'
```

Comment:
I think it's negligible post-gzip, yeah. Ultimately the proper fix would be to move the mcc/mnc to the carriers module I created in marketplace-constants but that was a lot of work that didn't bring much improvement so I decided against it for this PR.